Shift3/boilerplate-client-react
# Added `delete loading spinner` to `confirmation modal`

## Changes
Added a loading spinner to the delete button by adding the `LoadingButton` component.

## Purpose
A loading spinner was added when making an API call to delete a user, agency, or agent.

## Pre-Testing TODOs
Make sure to run the `Node Server`.

## Testing
1. Pull in the changes to your local copy of this branch and run `yarn star`.
2. Log in and navigate to any of the list views: `AgencyListView` `AgentListView` or `UserListView`.
3. Delete a user, agency, or agent and verify that a loading spinner appears. 

### Closes #509
